client/src/panels/panel_declarations/finances/vote_stat/vote_stat_text.yaml

```diff
@@ -51,28 +51,25 @@ gov_in_year_voted_stat_split_text:
   transform: [handlebars, markdown]
   en: |
     As of the {{est_in_year}} {{last_estimates}}, 
-    the government has presented Parliament with planned {{gl_tt "budgetary" 'BUD_EXP'}} expenditure estimates totaling **{{fmt_compact1_written gov_tabled_est_in_year}}**.
+    the government has presented Parliament with {{gl_tt "authorized" "AUTH"}} spendings totaling **{{fmt_compact1_written gov_tabled_est_in_year}}**.
     
-    Of those planned expenditures, **{{fmt_compact1_written gov_stat_est_in_year}}** (or **{{fmt_percentage1 gov_stat_percent_est_in_year}}**) 
+    Of those {{gl_tt "authorized" "AUTH"}} spendings, **{{fmt_compact1_written gov_stat_est_in_year}}** (or **{{fmt_percentage1 gov_stat_percent_est_in_year}}**) 
     will come from {{gl_tt "statutory authorities" "STAT"}} and **{{fmt_compact1_written gov_voted_est_in_year}}** (or **{{fmt_percentage1 gov_voted_percent_est_in_year}}**) 
     will come from {{gl_tt "voted authorities" "VOTED"}}.
 
   fr: |
-    En date du {{last_estimates}} {{est_in_year}}, le gouvernement a présenté au Parlement des prévisions de dépenses totalisant **{{fmt_compact1_written gov_tabled_est_in_year}}**. 
-    
-    De ces dépenses prévues, **{{fmt_compact1_written gov_stat_est_in_year}}** (ou **{{fmt_percentage1 gov_stat_percent_est_in_year}}**) proviendront des {{gl_tt "autorisations législatives" "STAT"}}
-    et **{{fmt_compact1_written gov_voted_est_in_year}}** (ou **{{fmt_percentage1 gov_voted_percent_est_in_year}}**) des {{gl_tt "autorisations votées" "VOTED"}}.
+    TODO
 
 dept_in_year_voted_stat_split_text:
   transform: [handlebars,markdown]
   en: |
-   Of the **{{fmt_compact1_written dept_tabled_est_in_year_estimates}}** estimated expenditures for 
-   **{{subject.name}}**, **{{fmt_percentage1 dept_stat_percent_est_in_year}}** (or **{{fmt_compact1_written dept_stat_est_in_year}}**) 
+   Of the **{{subject.name}}**'s **{{fmt_compact1_written dept_tabled_est_in_year_estimates}}** in {{gl_tt "authorized" "AUTH"}} spendings, 
```

Comment:
Maybe
"Of the {{value}} total authorized spending available to {{name}} in {{year}}, {{value}} (or {{percent}}) will be funded..."